pkg/console/subresource/deployment/deployment.go

```diff
@@ -334,3 +340,17 @@ func IsAvailableAndUpdated(deployment *appsv1.Deployment) bool {
 		deployment.Status.ObservedGeneration >= deployment.Generation &&
 		deployment.Status.UpdatedReplicas == deployment.Status.Replicas
 }
+
+func AddVolumeForCustomLogo(operatorConfig *operatorv1.Console, volConfigList []volumeConfig) []volumeConfig {
+	if logoFile := operatorConfig.Spec.Customization.CustomLogoFile.Name; logoFile != "" {
+		logoVolume := volumeConfig{
+			name:        logoFile,
+			readOnly:    true,
+			path:        "/app/" + operatorConfig.Spec.Customization.CustomLogoFile.Key,
```

Comment:
This shouldn't include the filename.

```suggestion
			path:        "/var/logo",
```